config/config.go

```diff
@@ -118,6 +118,7 @@ func init() {
 
 	cfg.SetDefault("flow.expire", 600)
 	cfg.SetDefault("flow.update", 60)
+	cfg.SetDefault("flow.hold", 15)
```

Comment:
could you set flow.how as a constant in flow.go like const HoldTimeout
don't need to be a parameter as it's not to be attend to change